lessons/2 - git time/index.html

```diff
@@ -1,80 +1,294 @@
+        }
 
-		.reveal li pre {
-			width: fit-content;
-		}
+        .reveal li pre {
+            width: fit-content;
+        }
 
-		.logo {
-			position: fixed;
-			/* Sit on top of the page content */
-			z-index: 2;
-			/* Specify a stack order in case you're using a different order for other elements */
-		}
-	</style>
+        span.exe {
+            font-weight: bold;
+            font-style: oblique;
+            color: #c9c9c9;
+        }
+
+        .logo {
+            position: fixed;
+            /* Sit on top of the page content */
+            z-index: 2;
+            /* Specify a stack order in case you're using a different order for other elements */
+        }
+    </style>
 </head>
 
 <body>
-	<a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo" /></a>
-	<div class="reveal">
-		<div class="slides">
-			<section>
-				<h1>Lecture</h1>
-				<h3>Git time</h3>
-			</section>
-		</div>
-	</div>
+<a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo"/></a>
+<div class="reveal">
+    <div class="slides">
+        <section>
+            <h1>Lecture</h1>
+            <h3>Git time</h3>
+        </section>
+        <section>
+            <h3>Create git hub account</h3>
+            <ul>
+                <li class="fragment">Open https://github.com/</li>
+                <li class="fragment">Sign up: username, email, password</li>
+                <li class="fragment">Choose free option</li>
+                <li class="fragment">Verify email - click link in email</li>
+                <li class="fragment">Done</li>
+            </ul>
+        </section>
+        <section>
+            <h3>Create git project</h3>
+            <ul>
+                <li class="fragment">Open https://github.com/ and login if needed</li>
+                <li class="fragment">Select 'repositories' and create new one</li>
+                <li class="fragment">Name it redi_js_spring_2020</li>
+                <li class="fragment">Make it public</li>
+                <li class="fragment">Done</li>
+            </ul>
+        </section>
+        <section>
+            <h3>Invite collaborators</h3>
+            <ul>
+                <li class="fragment">Open https://github.com/</li>
+                <li class="fragment">Select your project, open settings</li>
+                <li class="fragment">Select: Manage access</li>
+                <li class="fragment">Add all teachers as collaborators:</li>
+                <ul class="fragment">
+                    <li>aberezhk, BTBTravis, caritosteph, jeffersondarcy, MrTim</li>
+                </ul>
+                <li class="fragment">Done</li>
+            </ul>
+        </section>
+        <section>
+            <h3>Download and install git</h3>
+            <ul>
+                <li class="fragment">Check if you have 64 or 32 bit system</li>
+                <li class="fragment">Open https://git-scm.com/download/win</li>
```

Comment:
maybe https://git-scm.com/download because could be Mac users

Reply:
We can create the slides showing them how to install git in different OS like this [tutorial](https://www.atlassian.com/git/tutorials/install-git) but just with the necessary information or share with them the link.

Reply:
@caritosteph nice tutorial! And I agree that we can grab info from there and put into separate slides for MAC and Windows users. And provide this link so students can read about git, version control and so on.

Reply:
Changed link as @MrTim suggested. Added link to atlassian tutorial in the end of slides deck